content/10-minute-tutorial.md

```diff
@@ -1,3 +1,328 @@
+
+```
+├── pom.xml
+└── src
+    ├── main
+    │   └── java        (marked as sources root)
+    │   └── resources   (marked as resources root)
+    └── test
+        └── java        (marked as test sources root)
+```
+
+## Add Cucumber to your project
+
+Add Cucumber to your project by adding a dependency to your `pom.xml`:
+
+```xml
+<dependencies>
+    <dependency>
+        <groupId>io.cucumber</groupId>
+        <artifactId>cucumber-java</artifactId>
+        <version>{{% version "cucumberjvm" %}}</version>
+        <scope>test</scope>
+    </dependency>
+</dependencies>
+```
+
+In addition, you will need the following dependencies to run Cucumber with JUnit:
+```xml
+<dependencies>
+    ...
+    <dependency>
+        <groupId>io.cucumber</groupId>
+        <artifactId>cucumber-junit</artifactId>
+        <version>{{% version "cucumberjvm" %}}</version>
+        <scope>test</scope>
+    </dependency>
+
+    <dependency>
+        <groupId>junit</groupId>
+        <artifactId>junit</artifactId>
+        <version>4.12</version>
+        <scope>test</scope>
+    </dependency>
+</dependencies>
+```
+
+If you have IntelliJ configured to autoimport dependencies, it will automatically import them for you.
+Otherwise, you can manually import them by opening the "**Maven Projects** menu on the right and clicking the **Reimport all Maven Projects** icon on the top left of that menu.
+To check if your dependencies have been downloaded, you can open the **External Libraries** in the left **Project** menu in IntelliJ.
+
+To make sure everything works together correctly, open a command prompt and navigate to your project directory (the one containing the pom.xml file) and enter `mvn clean test`.
+
+You should see something like the following:
+```
+[INFO] Scanning for projects...
+[INFO]
+[INFO] ------------------------------------------------------------------------
+[INFO] Building cucumber-tutorial 1.0-SNAPSHOT
+[INFO] ------------------------------------------------------------------------
+
+    <progress messages....>
+
+[INFO] ------------------------------------------------------------------------
+[INFO] BUILD SUCCESS
+[INFO] ------------------------------------------------------------------------
+[INFO] Total time: <time> s
+[INFO] Finished at: <date> <time>
+[INFO] Final Memory: <X>M/<Y>M
+[INFO] ------------------------------------------------------------------------
+
+```
+
+Your project builds correctly, but nothing is tested yet as you have not specified any behaviour to test against.
+
+# Specifying Expected Behaviour
+
+We specify the expected behaviour by defining features and scenarios.
+
+## Creating the Feature Directory
+
+Features are defined in feature files, which are stored in the `src/test/resources/` directory (or a subdirectory).
+
+We need to create this directory, as it was not created for us. In IntelliJ:
+
+1. Right click on the Test folder, select **New > Directory** and name this directory `resources`.
```

Comment:
I'd probably skip telling people how to create a dir, and just tell them how to mark it as Test Resources Root